package.json

```diff
@@ -126,16 +126,16 @@
     "gen:parsers": "yarn --cwd packages/kas gen:parsers",
     "build": "yarn gen:parsers && rollup -c config/build/rollup.config.js",
     "build:types": "yarn tsc --build tsconfig-build.json",
-    "build:flowtypes": "SWCRC=true node -r @swc-node/register config/build/gen-flow-types.ts",
+    "build:flowtypes": "config/build/gen-flow-types.ts",
```

Comment:
Cool. I like that these are simpler. That'll mean they'll work correctly if just executed from the shell too!